src/main/java/frc/robot/Constants.java

```diff
@@ -16,4 +16,29 @@ public final class Constants {
   public static class OperatorConstants {
     public static final int kDriverControllerPort = 0;
   }
+
+  public static class IntakeConstants {
+    //TODO: Assign a real CAN ID
+    public static final int kIntakeMotorID = 0;
+  }
+
+  public static class DriveConstants {
+    public static final int kSmartCurrentLimit = 40;
+  }
+
+  public static class ShooterConstants {
+    //TODO: Assign real CAN ID's for the 2 flywheels
+    public static final int kLeftFlywheelID = 0;
+    public static final int kRigthFlywheelID = 0;
+
+    public static final double kDriveEncoderCPR = 2048.0;
```

Comment:
https://github.com/keira1008/Thor_2023/blob/465fcfb916edc36d50df0602baf779c7dc221336/src/main/java/frc/robot/Constants.java#L142 

This was recently changed.